### PR TITLE
Update docs dependencies

### PR DESCRIPTION
**Pull request by joelspadin**
Updated dependencies for the docs website.

Newer versions of React and MDX are available, but they aren't supported by Docusaurus yet.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
```
[the remark this answers is not in the document]
Comment (joelspadin) [reply package-lock.json:1]: Whoops. I must have accidentally run an npm command from the top level at some point and it created that.

File: package.json [hunk 2]
```diff
@@ -0,0 +1 @@
+{}
```
[the remark this answers is not in the document]
Comment (joelspadin) [reply package.json]: Fixed

File: package-lock.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (joelspadin) [reply package-lock.json]: Fixed.

**Review by petejohanson — COMMENTED** [review]
Is adding a compiled wasm really the best we can do?

**Comment by joelspadin** [comment]
> Is adding a compiled wasm really the best we can do?

For the moment, yes... Until https://github.com/tree-sitter/tree-sitter/pull/2183 gets released, it is very difficult to compile the wasm file on Windows. Once that is included in a release, I can look into compiling the wasm file as part of the docs build (though that will require you to have Docker or Emscripten installed to build the docs).

**Review by petejohanson — APPROVED** [review]
Thankfully the binary warm is fairly small. So be it.

**Pull request merged**